components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -183,7 +183,7 @@
 FASTCGITCP = "fastcgi-tcp"
 FASTCGI_TYPES = (FASTCGITCP, )
 DEVELOPMENT = "development"
-DEFAULT_SERVER_TYPE = FASTCGITCP
+DEFAULT_SERVER_TYPE = WSGITCP
```

Comment:
This default change is my primary concern on this PR. While it makes sense on the 5.2.x line, this will likely cause most 5.1.x Web instances to fail at deployment after 5.1.4 upgrade like https://ci.openmicroscopy.org/job/OMERO-5.1-merge-deploy/584/ since the server and/or the nginx configuration will need to be reviewed/regenerated.
Would it be possible to maintain fastcgi-tcp as the default value but keep the deprecation warning suggesting to migrate to wsgi/gunicorn as on opt-in for 5.1.x?